### DISCUSSION TITLE
Porosity and pore pressure sign convention

### DISCUSSION THREAD

**Discussion by jmeier** [idaholab/moose#29182] [Q&A Modules: Porous Flow] [answered]
Dear Moose community,

I'd like to double-check two things with you:

**First, sign convention in SoildMechanics and PorousFlow:**

In #20999, some info on the sign convention in SoildMechanics (formerly TensorMechanics) is given:

> It is the same convention as TensorMechanics, which is:
> - a positive strain is extensional, which (usually) results from a positive (tensile) stress;
> - a negative strain is compressional, which (usually) results from a negative (compressive) stress

Just to be sure, I'd like to spell it out for the pore-pressures:
- Compressive pore-pressures (e.g. below a free water table) have a positive sign?
- Suction is represented by pore-pressures having a negative sign?

Is this correct?

**Second, the physical unit of the permeability**

`PorousFlowPermeabilityConst` allowes to input a "permeability"-tensor (internally stored in `PorousFlowPermeabilityBase._permeability_qp`). It is correct, that the physical unit of the items of this permeability-tensor is "length to the power of two" (e.g. squaremeters, m²)?

Thanks for you help in advance.

**Comment by GiudGiud** [comment]
@cpgr

**Reply by GiudGiud** [reply]
or @1runer ?

**Comment by cpgr** [comment, marked as answer]
Yes and yes (mostly). The units will be m^2 if everything else uses 'typical' units like Pa, kg/m^3 etc
